app/services/feed_events/bulk_upsert.rb

```diff
@@ -57,6 +57,7 @@ def call
       end
 
       FeedEvent.insert_all(records_to_insert) if records_to_insert.present?
+      FeedEvent.bulk_update_counters_by_article_id(records_to_insert.pluck(:article_id).sample(5))
```

Comment:
I'm not sure if this matters, but the line before this accounts for when `if records_to_insert.present?` Should the same be done here?

Reply:
Good call.